setup.py

```diff
@@ -65,10 +66,26 @@ def get_version():
     raise RuntimeError(f"Unable to find version in {VERSIONFILE}.")
 
 
+VERSION = get_version()
```

Comment:
Can the function `get_version()`  be updated to take `BUILD_PYMC_NIGHTLY` as a parameter? Its  odd to have this secondary function that gets defined under an if block and a global thats overwritten